spec/features/user_can_create_a_conversation_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "rails_helper"
+
+feature 'User can create a new message' do
```

Comment:
In this test you are only looking at if there are some input fields. I think it would better to test if a user can actually send off a message. I think the "Warden helpers & Mailboxer methods" document would help you out with that.